src/Status.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 const Status = ({ item }) => {
   const { completed } = item;
+  const className = classNames({
+    completed: completed === true,
+    not__completed: completed === false,
+  });
 
   return (
     <>
-      {completed ? <p className="completed">completed</p>
-        : <p className="not__completed">not completed</p>}
+      {completed ? <p className={className}>completed</p>
```

Comment:
функции можно вызывать прям тут
```suggestion
      {completed ? <p className={classNames({
    completed: completed === true,
    not__completed: completed === false,
  })}>completed</p>
```